src/YQWizard.cc

```diff
@@ -1097,6 +1106,49 @@ void YQWizard::showHelp()
 }
 
 
+void YQWizard::showHotkeys()
+{
+    /**
+     * Help text to be shown after pressing Shift-F1 listing the advanced
+     * keyboard shortcuts available in the Qt-UI
+     **/
+    _qHotkeysText = _(
+        "<h1>Advanced Hotkeys</h1>"
+        "<dl>"
+        "<dt>Ctrl-Alt-Shift-X</dt>"
+        "<dd>open a terminal window (xterm), useful for VNC installs</dd>"
+        "<dt>Shift-F7</dt>"
+        "<dd>Debug level</dd>"
+        "<dt>Ctrl-Shift-D</dt>"
+        "<dd>Debug module write</dd>"
+        "<dt>Print key</dt>"
+        "<dd>Screenshot</dd>"
```

Comment:
what screenshot? What it do? Store screenshot somewhere? and where? or open dialog where to store screenshot?